spock-specs/src/test/groovy/org/spockframework/docs/datadriven/v4/MathSpec.groovy

```diff
@@ -1,11 +1,11 @@
 package org.spockframework.docs.datadriven.v4
 
+import spock.lang.Rollup
 import spock.lang.Specification
-import spock.lang.Unroll
 
 class MathSpec extends Specification {
 // tag::example[]
-  @Unroll
+  @Rollup
```

Comment:
Because this is a documentation example and the example for the documentation changed?
Before the documentation said "everything is uprolled, use `@Unroll` to unroll, example:". Now it says "everything is unrolled, use `@Rollup` to uproll, example:".